src/style.css

```
@@ -0,0 +1,85 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif ;
+  margin: 0;
+
+}
+
+.link,
+.section,
+.list {
+  margin: 0;
+  padding: 0;
+}
+
+.hat {
+  height: 60px;
+  padding: 10px 50px;
+  margin: 0;
+  background-color: rgb(255, 255, 255);
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  
+  
+}
+
+.nav {
+  font-size: 12px;
+  font-weight: 500;
+  color: rgb(0 0 0);

```

Коли задаєш колір у форматі RGB, розділяй значення комами.
```suggestion
  color: rgb(0,0,0);
```

---

Извини за невнимательность.